py/server/deephaven/experimental/iceberg.py

```diff
@@ -399,10 +401,14 @@ def __init__(self,
                 `None`, which means use 2^20 (1,048,576)
             target_page_size (Optional[int]): the target Parquet file page size in bytes, if not specified. Defaults to
                 `None`, which means use 2^20 bytes (1 MiB)
-            sort_order_provider: Optional[SortOrderProvider]: Used to provide SortOrder to be used for sorting new data
-                while writing to an iceberg table using this writer. Note that we select the sort order of the Table at
-                the time the writer is constructed, and it does not change if the table's sort order changes. Defaults
-                to `None`, which means use the table's default sort order.
+            sort_order_provider (Optional[SortOrderProvider]): Specifies the sort order to use for sorting new data
+                when writing to an Iceberg table with this writer. The sort order is determined at the time the writer
+                is created and does not change if the table's sort order changes later. Defaults to `None`, which means
+                the table's default sort order is used. More details about sort order can be found in the Iceberg
+                spec: https://iceberg.apache.org/spec/#sorting
```

Comment:
Does the link need to be notated in any special way to get a hyperlink in sphinx?